config/foreman.migrations/20190111180118_delete_removed_settings.rb

```diff
@@ -0,0 +1,63 @@
+DELETED = {
+  'foreman': [
+    'custom_repo',
+    'dynflow_in_core',
+    'email_conf',
+    'email_config_method',
+    'email_source',
+    'puppet_home',
+    'puppet_ssldir',
+  ],
```

Comment:
Should we also add `authentication`, `locations_enabled` and `organizations_enabled` here?

Reply:
They should be in the clear section since the parameters still exist